webscrapers/covents_json.json

```diff
@@ -0,0 +1,142 @@
+{
+  "1": {
```

Comment:
Hi, are you storing these id's anywhere?  I think when we add these objects into the DB, the DB will assign them for us, but if these are unique id's that we've pulled from somewhere, they would be useful to store in the DB as a check to see if it already exists before we add them.  If they are not unique (same 20 ids are created each time we scrape, then they might not be necessary, however, I don't think they do any harm to have them.

Reply:
So I can remove the ids if we want. I can make the keys of each object a number or the event name. I figured the incrementing number is easier to read as some event names are huge. What are your thoughts FE and Kelly

Reply:
Well, the database will assign those id's, but it doesn't hurt to have them in here.

Reply:
Ok sounds good!